### ISSUE TITLE
Version 64 locks the apps grid for changes.

### ISSUE THREAD

**Issue by eitanmi** [icedman/dash2dock-lite#128]
Tested on Fedora 40  / Gnome 46.

**Comment by icedman** [comment]
This one is related to #126 .
Please try the development branch for the fix. I didn't upload to EGO yet as the fix still has to be check with multi-monitor setups.

**Comment by eitanmi** [comment]
Yep, looks like like development version 100 (?)  resolves this.

**Comment by HollowMoon477** [comment]
![image](https://github.com/icedman/dash2dock-lite/assets/104750233/412ecd8d-193d-45ee-9abf-e246ad00b8d2)
My problem still exists
